src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -578,9 +578,45 @@ private function addWorkflowSection(ArrayNodeDefinition $rootNode): void
                                                         ->ifArray()
                                                         ->then(static fn ($from) => array_map(static fn ($v) => $v instanceof \BackedEnum ? $v->value : $v, $from))
                                                     ->end()
+                                                    ->beforeNormalization()
+                                                        ->always()
+                                                        ->then($workflowNormalizeArcs = static function ($arcs) {
+                                                            $arcs = (array) $arcs;
```

Comment:
acceptAndWrap() above already did the cast:
```suggestion
                                                        ->ifArray()
                                                        ->then($workflowNormalizeArcs = static function ($arcs) {
```